package.json

```diff
@@ -23,6 +23,7 @@
   "author": "Orta",
   "license": "MIT",
   "dependencies": {
+    "@types/body-parser": "^1.16.4",
```

Comment:
Do we need to ship all the `@types` packages as dependencies, or can they be moved to devDependencies?